tools-public/toolpad/pages/validatePrioritySupport/page.yml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
+kind: page
+spec:
+  id: jyhs86t
+  title: validatePrioritySupport
```

Comment:
I think that it should be called:

```suggestion
  title: validateSupport
```

but I'm unable to make this change. When I edit it manually in the yml it magically reverts back. When I try to edit it in the editor, it fails with:

<img width="1277" alt="Screenshot 2023-06-12 at 18 16 21" src="https://github.com/mui/mui-public/assets/3165635/262114cc-a263-4254-b24f-935be02ad015">

cc @Janpot

Reply:
👍  Will take a look tomorrow.
@prakhargupta1 I remember you've ran into this but I can't find a ticket. Is there a ticket?

Reply:
Yes, this one: https://github.com/mui/mui-toolpad/issues/2072

Reply:
> Yes, this one: https://github.com/mui/mui-toolpad/issues/2072

Ah yes, two errors in one 🎁